core/tests/protractor/libraryPagesTour.js

```diff
@@ -0,0 +1,76 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview End-to-end tests to check for console errors in
+ * library related pages
+ */
+
+var general = require('../protractor_utils/general.js');
+var admin = require('../protractor_utils/admin.js');
+var library = require('../protractor_utils/library.js');
+var player = require('../protractor_utils/player.js');
+var users = require('../protractor_utils/users.js');
+
+describe('Library pages tour', function() {
+  var EXPLORATION_TITLE = 'About Oppia';
+  var EXPLORATION_RATING = 4;
+  var SEARCH_TERM = 'python';
+  var visitLibraryPage = function() {
+    browser.get(general.LIBRARY_URL_SUFFIX);
+  };
+
+  it('adds a rating to an existing exploration', function() {
+    users.createAndLoginAdminUser('randomuser@gmail.com', 'r4nd0m');
+    var profileDropdown = element(by.css('.protractor-test-profile-dropdown'));
```

Comment:
In this case, you aren't testing how the user gets to the admin page, so I think you can just do

    browser.get(general.ADMIN_URL_SUFFIX);

But you don't need these three lines anyway, right? The get request is already part of admin.reloadAllExplorations(), so I would just follow the same design pattern as other protractor tests that currently use admin.reloadExploration().